traffic_portal/app/src/common/modules/navigation/navigation.tpl.html

```diff
@@ -52,6 +52,7 @@
                 </li>
                 <li class="side-menu-category"><a href="javascript:void(0);"><i class="fa fa-sm fa-chevron-right"></i> Topology</span></a>
                     <ul class="nav child_menu" style="display: none">
+<!--                        <li class="side-menu-category-item" ng-if="hasCapability('topologies-read')" ng-class="{'current-page': isState('trafficPortal.private.topologies')}"><a href="/#!/topologies">Topologies</a></li>-->
```

Comment:
Do we want this commented out?

Reply:
yeah, it's my way of hiding the feature from TP until everything is good to go.

Reply:
it will get uncommented soon enough and definitely before 5.0